components/App.tsx

```diff
@@ -250,13 +249,15 @@ export function App(props: Props) {
         <QueryClientProvider client={queryClient}>
           <ThemeProvider defaultTheme="Dark">
             <WalletIdentityProvider appName={'Realms'}>
+              <WalletProvider>
```

Comment:
i think the next line(s) need to be indented

Reply:
Done!